chat.go

```diff
@@ -63,6 +74,17 @@ func NewPostMessageParameters() PostMessageParameters {
 	}
 }
 
+// NewPostEphemeralParameters provides an instance of PostEphemeralParameters with all the sane default values set
+func NewPostEphemeralParameters() PostEphemeralParameters {
+	return PostEphemeralParameters{
+		User:        DEFAULT_MESSAGE_USERNAME,
+		AsUser:      DEFAULT_MESSAGE_ASUSER,
+		Attachments: nil,
```

Comment:
The API supports attachments in ephemeral messages right? Any chance we can get that in, and perhaps try to re-use the `postMessage` as much as possible (looks like the text support the same attributes as text in a normal message)?

Reply:
This is just the default, setting attachments to `nil` to start. You can (and we do) set attachments in these post types for sure, you just have to allocate that separately in your use of the struct

Reply:
I apologise, I wasn't clear. I was trying to say this: ephemeral messages seem to support much the same features as normal messages and so I think we should support them in the API as well. In other words: let's try to re-use as much code as possible from the normal message code paths. That way we'll get all the unfurling, attachment and text formatting options "for free". Do you think that is possible?

Reply:
Sure, can try this and get back to you.